git_parser.py

```diff
@@ -43,4 +50,5 @@ def show_commit(url):
 
     wb.close()
 
-
+if __name__=='__main__':
+    show_commit(args.output_file, args.repository)
```

Comment:
Pass args as is - what if we would like to add more flags?
Always thing further uses... we need to write scalable code.